src/components/shared/ConstellationCard.vue

```diff
@@ -20,7 +20,7 @@ export default {
 </script>
 
 <style scoped lang="scss">
-    @import '../styles/colors';
+    @import '../../styles/colors';
```

Comment:
Should we include `colors.scss` globaly?

Reply:
I meant to look into this more. 

It wouldn't work if we globally declare it the same way as the typography, since the colors have variables in it. Seems like this would be a solution
https://css-tricks.com/how-to-import-a-sass-file-into-every-vue-component-in-an-app/